### DISCUSSION TITLE
vertical bandwidth scale/chroma smearing

### DISCUSSION THREAD

**Discussion by Qirlfriend** [ntsc-rs/ntsc-rs#205] [Ideas]
the way the colors smear out and create blurry blobs is so realistic and nice, but it only seems to happen horizontally, i think it might add something if there was a way to smear the chroma in both directions somehow, or maybe that's not how it works and I just don't know enough about VHS but look at how sharp the borders are when you adjust the vertical chroma delay vs. horizontal.  I dunno, is that how VHS's work and i never noticed?
![SMPTE_Color_Bars svg_ntsc](https://github.com/user-attachments/assets/836b8796-6d6c-4553-badf-100f51b9ce87)

**Comment by valadaptive** [comment]
Real VHS tapes/analog TV broadcasts do have limited vertical resolution, but the analog signal itself is 1D--each row comes one after the other, so they don't blur together. The best way to emulate this is to downscale the footage to 480p before applying the effect. There's an option for this in the standalone application, or you can do so in whatever editing software you're using (making sure that the downscaling happens *before* the ntsc-rs effect).

**Reply by Qirlfriend** [reply]
That makes sense, maybe what i'm thinking of is general video compression since I've only ever seen them through youtube
